cinema/views.py

```diff
@@ -81,12 +101,16 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         MovieSession.objects.all()
         .select_related("movie", "cinema_hall")
         .annotate(
-            tickets_available=F("cinema_hall__rows")
-            * F("cinema_hall__seats_in_row")
-            - Count("tickets")
+            tickets_available=(
+                F("cinema_hall__rows")
+                * F("cinema_hall__seats_in_row")
+                - Count("tickets")
+            )
```

Comment:
The calculation of `tickets_available` seems correct, but ensure that the logic aligns with the business requirements. Double-check that subtracting the count of tickets from the product of rows and seats in a row is the intended calculation.